lib/poetry.js

```diff
@@ -32,9 +32,26 @@ function pyprojectTomlToRequirements() {
   if (res.status !== 0) {
     throw new Error(res.stderr);
   }
+
+  const editableLine = new RegExp(/^-e /gm);
+  const sourceRequirements = path.join(this.servicePath, 'requirements.txt');
+  const requirementsContents = fse.readFileSync(sourceRequirements, {
+    encoding: 'utf-8'
+  });
+
+  if (requirementsContents.match(editableLine)) {
+    this.serverless.cli.log(
+      'The generated file contains -e lines, removing them...'
+    );
+    fse.writeFileSync(
+      sourceRequirements,
+      requirementsContents.replace(editableLine, '')
```

Comment:
Would it be more accurate to call this variable `ediableFlag` instead of `editableLine`? This made me think the whole requirement is being deleted, which isn't right, afaict

```suggestion
      requirementsContents.replace(editableFlag, '')
```